### ISSUE TITLE
fail positive on github action fails.

### ISSUE THREAD

**Issue by gengjiawen** [nodejs/node-core-utils#660]
<details>

<summary>Commit Queue failed</summary><pre>- Loading data for nodejs/node/pull/45573
✔  Done loading data for nodejs/node/pull/45573
----------------------------------- PR info ------------------------------------
Title      deps: V8: cherry-pick 2ada52cffbff (#45573)
Author     Michaël Zasso <targos@protonmail.com> (@targos)
Branch     targos:fix-45171 -> nodejs:main
Labels     build, v8 engine, needs-ci
Commits    1
 - deps: V8: cherry-pick 2ada52cffbff
Committers 1
 - Michaël Zasso <targos@protonmail.com>
PR-URL: https://github.com/nodejs/node/pull/45573
Fixes: https://github.com/nodejs/node/issues/45171
Refs: https://github.com/v8/v8/commit/2ada52cffbff11074abfaac18938bf02d85454f5
Reviewed-By: Jiawen Geng <technicalcute@gmail.com>
Reviewed-By: Richard Lau <rlau@redhat.com>
Reviewed-By: Colin Ihrig <cjihrig@gmail.com>
Reviewed-By: Yagiz Nizipli <yagiz@nizipli.com>
------------------------------ Generated metadata ------------------------------
PR-URL: https://github.com/nodejs/node/pull/45573
Fixes: https://github.com/nodejs/node/issues/45171
Refs: https://github.com/v8/v8/commit/2ada52cffbff11074abfaac18938bf02d85454f5
Reviewed-By: Jiawen Geng <technicalcute@gmail.com>
Reviewed-By: Richard Lau <rlau@redhat.com>
Reviewed-By: Colin Ihrig <cjihrig@gmail.com>
Reviewed-By: Yagiz Nizipli <yagiz@nizipli.com>
--------------------------------------------------------------------------------
   ⚠  Commits were pushed since the last review:
   ⚠  - deps: V8: cherry-pick 2ada52cffbff
   ℹ  This PR was created on Tue, 22 Nov 2022 08:21:45 GMT
   ✔  Approvals: 4
   ✔  - Jiawen Geng (@gengjiawen): https://github.com/nodejs/node/pull/45573#pullrequestreview-1189584648
   ✔  - Richard Lau (@richardlau) (TSC): https://github.com/nodejs/node/pull/45573#pullrequestreview-1190006137
   ✔  - Colin Ihrig (@cjihrig) (TSC): https://github.com/nodejs/node/pull/45573#pullrequestreview-1190051320
   ✔  - Yagiz Nizipli (@anonrig): https://github.com/nodejs/node/pull/45573#pullrequestreview-1191762773
   ✖  Last GitHub CI failed
   ℹ  Last Full PR CI on 2022-11-23T14:48:53Z: https://ci.nodejs.org/job/node-test-pull-request/48124/
- Querying data for job/node-test-pull-request/48124/
   ✔  Last Jenkins CI successful
--------------------------------------------------------------------------------
   ✔  Aborted `git node land` session in /home/runner/work/node/node/.ncu</pre><a href='https://github.com/nodejs/node/actions/runs/3538908052'>https://github.com/nodejs/node/actions/runs/3538908052</a></details>

_Originally posted by @nodejs-github-bot in https://github.com/nodejs/node/issues/45573#issuecomment-1326111158_

**Comment by targos** [comment]
It's not a false positive. I rebased and pushed after the last review.

**Comment by targos** [comment]
Yes, but that's expected. Someone needs to submit a review after the push.

**Comment by targos** [comment]
Isn't it because of this?
![image](https://user-images.githubusercontent.com/2352663/203738569-01e7036d-af3d-4ecc-b996-fad01fe62663.png)

**Comment by targos** [comment]
I reopened because the error was wrong, but I still think the failure was expected (Jenkins CI had a failure).

**Comment by aduh95** [comment]
Yeah, "GitHub CI" is not very accurate because all it does is check the "status of the PR", which includes the Jenkins results.

**Comment by targos** [comment]
But why did ncu say "Last Jenkins CI successful" when https://ci.nodejs.org/job/node-test-pull-request/48124/ is clearly red?

**Comment by gengjiawen** [comment]
Another one: https://github.com/nodejs/node-core-utils/issues/691

**Comment by targos** [comment]
I had a look and it's because of this part of the code:
https://github.com/nodejs/node-core-utils/blob/4ef0fe2607f3986f702bcd1ada244965cfcd4457/lib/pr_checker.js#L393-L410

It's actually the Jenkins CI check that is red. I wonder if we should remove this whole block. I don't know what kind of "old" checks we expect to be here.

![CleanShot 2023-04-03 at 14 09 12](https://user-images.githubusercontent.com/2352663/229504820-1a9df159-6411-49b8-bfa2-b660041cab06.png)
